test/integration/CMakeLists.txt

```diff
@@ -29,4 +29,21 @@ endif()
 
 add_subdirectory(plugin)
 
-unset( PLUGIN_BACKENDS )
+if(NOT WIN32)
+raja_add_test(
+  NAME test-plugin-dynamic
+  SOURCES test_plugin_dynamic.cpp)
+
+add_library(dynamic_plugin SHARED plugin_for_test_dynamic.cpp)
```

Comment:
Can you replace this with blt_add_library (https://llnl-blt.readthedocs.io/en/develop/api/target.html#blt-add-library), and make it "depend" on hip. You should be able to remove the target_link_libraries line.